src/argument/ArgumentResolversBuilderImpl.ts

```diff
@@ -21,8 +21,8 @@ class ArgumentResolversBuilderImpl extends AbstractBuilderImpl<ArgumentsResolver
 		return this;
 	}
 
-	withInstanceId(): ArgumentsResolversBuilder {
-		this.getInstance().add(new InstanceIdArgumentResolver());
+	withInstanceId(count: number = 1): ArgumentsResolversBuilder {
```

Comment:
Would it be more semantically clear to have two different builder methods and resolvers for the single id case and the multiple ids case (withInstanceId() and withInstanceIds() respectively) in order to not have any magic happen with regards to what is injected into the constructor argument?